### PR TITLE
XEP-0420: Version 0.2

### PR DESCRIPTION
**Pull request by vanitasvitae**
This PR introduces IQ encryption workflow and adds missing explanation and examples.

### CODE REVIEW

**Comment by vanitasvitae** [comment]
Let me know if I shall squash the commits together :)

File: xep-0420.xml [hunk 1]
```diff
@@ -35,6 +35,15 @@
     <email>vanitasvitae@fsfe.org</email>
     <jid>vanitasvitae@jabberhead.tk</jid>
   </author>
+  <revision>
+    <version>0.1.1</version>
```
Comment (horazont) [inline xep-0420.xml]: @vanitasvitae These are not editorial changes, you need to go to 0.2.0.

**Comment by horazont** [comment]
@vanitasvitae Assign back to me once you’re done.

**Comment by vanitasvitae** [comment]
@horazont I can't resassign you, but I changed the version code as requested. 👍

**Pull request merged**